part.go

```diff
@@ -393,21 +398,35 @@ func parseParts(parent *Part, reader *bufio.Reader) error {
 		// Look for part header.
 		bbr := bufio.NewReader(br)
 		if err = p.setupHeaders(bbr, ""); err != nil {
+			if skipMalformedParts {
+				parent.addError(ErrorMalformedChildPart, "read header: %s", err.Error())
```

Comment:
Could we could wrap the error with custom MultiPartError(err, p) so that if handling the error it would be possible to access it from outside and do any custom handling if needed, wdyt?

Reply:
like we can, but whole library using `.addError` style, I don't what to brake it